### DISCUSSION TITLE
Proxy support

### DISCUSSION THREAD

**Discussion by d41z3r** [jchristn/SuperSimpleTcp#128] [Q&A]
any way I can use proxy with client?

**Comment by jchristn** [comment]
Which proxy and what kind?

**Reply by d41z3r** [reply]
socks5 / residental

**Reply by jchristn** [reply]
It should be possible (see https://stackoverflow.com/questions/35066981/how-to-use-proxy-with-tcpclient-connectasync) - it's not currently supported in the library, but if you have a PR with this capability I'd be happy to review and add it!
